gluten-core/src/main/scala/io/glutenproject/GlutenPlugin.scala

```diff
@@ -73,6 +76,34 @@ private[glutenproject] class GlutenDriverPlugin extends DriverPlugin {
     BackendsApiManager.getContextApiInstance.shutdown()
   }
 
+  private def showGlutenBuildInfo(): Unit = {
+    val veloxInfo = if (veloxEnabled) {
```

Comment:
We can just use the below code to recognize velox backend:
`BackendsApiManager.getBackendName.equalsIgnoreCase(GlutenConfig.GLUTEN_VELOX_BACKEND)`

Reply:
thank you for the tip, I changed to use the new method